UndertaleModLib/Models/UndertaleRoom.cs

```diff
@@ -421,12 +427,30 @@ public class Background : UndertaleObject, INotifyPropertyChanged
             private UndertaleRoom _ParentRoom;
 
             /// <summary>
-            /// The room parent this background belongs to.
+            /// The room parent this background belongs to.<br/>
+            /// Set by <see cref="SetupRoom(bool)"/>.
```

Comment:
Unnecessary. If someone wants to see by what it's set, they can just "search all references." It's also not something that should be put into documentation.